legacy/package.json

```diff
@@ -24,7 +24,7 @@
     "prettier": "prettier --write 'src/**/*.{js,scss}' '!**/build.scss' '!**/*-min.js'",
     "prettier-check": "prettier --check 'src/**/*.{js,scss}' '!**/build.scss' '!**/*-min.js'",
     "prepublishOnly": "yarn build",
-    "start": "NODE_ENV=development webpack serve --config webpack.dev.js --port 8402 --host 0.0.0.0 --disable-host-check",
```

Comment:
I've consolidated all the config options in the config files themselves rather than have some of it exist here as well